order-routing/rust/rankers/default/Cargo.toml.liquid

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
It seems we don't have liquid in the default function app, why do we need them here?